scripts/converter/convert_transform_to_pose.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+# Copyright 1996-2023 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Convert Transforms with no scale or the default one into Poses"""
+
+# usage: python convert_transform_to_pose.py /path/to/your/webots/project
+from webots_parser import WebotsParser
+from pathlib import Path
+
+import sys
+
+
+def convert_to_enu(filename):
```

Comment:
This function name is a bit misleading, isn't it?
```suggestion
def convert_to_pose(filename):
```

Reply:
Ooops a bad copy/paste